src/scripts/map.js

```diff
@@ -1,99 +1,100 @@
+        precision: 0.1,
+        // translates the pixel coordinates of center based on array of [x, y]
+        translation: undefined,
+        // a class name or function that returns a class name to be set onto each feature
+        cssClass: undefined,
+        // a color (hex) or function that returns a color to be set onto each feature
+        fill: undefined
+    }
+};
 
-    /**
-    * Adds a map visualization to the Contour instance.
-    *
-    * This visualization requires the [`.geo()`](#geo) frame.
-    *
-    * Map visualizations require a TopoJSON file with the topology to draw. Because of this, typically map visualizations are created as part of a callback function passed to a `d3.json()` call that parses the TopoJSON file. 
-    *
-    * When you [download Contour-Geo](get_contour.html), a few TopoJSON files are included. You can also [create your own](#topojson).
-    *
-    * ### Example:
-    *
-    *       d3.json('us-states.json', function (us) {
-    *           new Contour({
-    *                   el: '.map',
-    *                   map: {
-    *                       projection: d3.geo.albersUsa()
-    *                   }
-    *               })
-    *               .geo()
-    *               .map(us)
-    *               .render();
-    *       });
-    *
-    *
-    * @name map(data, options)
-    * @param {object} data The data (topology) to be rendered with this visualization. This must be in TopoJSON format.
-    * @param {object} options (Optional) Configuration options particular to this visualization that override the defaults.
-    */
-    function map(data, layer, options) {
-        var width = options.chart.plotWidth;
-        var height = options.chart.plotHeight;
-        var opt = options.map || {};
-        var us = data;
-        var fillFn = !opt.fill ? _.noop : _.isFunction(opt.fill) ? opt.fill : function () { return opt.fill; };
-        var classFn = !opt.cssClass ? _.noop : _.isFunction(opt.cssClass) ? opt.cssClass : function () { return opt.cssClass; };
-        var scaleRatio = options.map.scaleRatio;
 
+/**
+* Adds a map visualization to the Contour instance.
+*
+* This visualization requires the [`.geo()`](#geo) frame.
+*
+* Map visualizations require a TopoJSON file with the topology to draw. Because of this, typically map visualizations are created as part of a callback function passed to a `d3.json()` call that parses the TopoJSON file.
+*
+* When you [download Contour-Geo](get_contour.html), a few TopoJSON files are included. You can also [create your own](#topojson).
+*
+* ### Example:
+*
+*       d3.json('us-states.json', function (us) {
+*           new Contour({
+*                   el: '.map',
+*                   map: {
+*                       projection: d3.geo.albersUsa()
+*                   }
+*               })
+*               .geo()
+*               .map(us)
+*               .render();
+*       });
+*
+*
+* @name map(data, options)
```

Comment:
Is the name used anywhere? The signature differs from the actual one here?

Reply:
The jsdoc entry is a reflection of how the visualization should be used and is for generation of the documentation, and not how the signature of the renderer function is defined... it may be a bit strange/confusing that is right before the definition of the renderer, but you'd not be calling that function directly.